packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -69,6 +69,9 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
   bool get enableButtons =>
       actionInProgress == false && connectionInProgress == false;
 
+  static const summaryTreeKey = Key('Summary Tree');
```

Comment:
This helps ensure inspector tree state widgets are reused when they should which avoids a known bug in the flutter scrollbars as well as helping performance a little.